### DISCUSSION TITLE
How to disable ArmouryCrateKeyControl on M4 key

### DISCUSSION THREAD

**Discussion by CNordy** [seerge/g-helper#330] [Q&A]
How do I disable the ArmouryCrateKeyControl popup from appearing when using the M4 key to open G-Helper? I've already run the official uninstaller for Armoury Crate and rebooted.

![image](https://user-images.githubusercontent.com/132780124/236640989-981e1151-fad3-4cf0-8518-c6aa9ed490a4.png)

![image](https://user-images.githubusercontent.com/132780124/236640999-de219bbc-74bf-45b8-a1e1-6340b29371d5.png)

**Comment by seerge** [comment]
I’ve heard that checking FAQ helps ;) https://github.com/seerge/g-helper#how-do-i-stop-the-armory-crate-install-popup-appearing-every-time-i-press-the-m4--rog-key

**Reply by CNordy** [reply]
Sorry, I missed this one...  _*runs away embarassed*_
